app/views/api/services/usage_rules.html.slim

```diff
@@ -0,0 +1,22 @@
+- # TODO: When APIAP is default, adjust the old settings cukes to usage rules
+- content_for :sublayout_title, 'Usage Rules'
```

Comment:
Hmm it would be safer to use it with `:flush` otherwise it will append to any previous content 
```suggestion
- content_for :sublayout_title, 'Usage Rules', flush: true
```